.github/workflows/publish-module-api-doc.yaml

```diff
@@ -28,12 +28,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  publish-api-doc:
-    # Run on push to the branch 'master' or on pull request if the 'full-ci'
-    # label is set.
-    if: github.repository == 'tarantool/tarantool' &&
```

Comment:
`github.repository == 'tarantool/tarantool'` condition looks meaningful, at least for publishing.